qemu/tests/ceph_image_mem_leak.py

```diff
@@ -0,0 +1,48 @@
+import time
+
+from avocado.utils import process
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    1) Start guest with 10 rbd data disks and system disk.
+    2) Check the block info for 2 hours
+    3) Check the used memory size is not increased
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+
+    # delay 2 minutes to wait the guest boot up completely
+    time.sleep(120)
```

Comment:
if you want to confirm if the guest boot up successfully, you may use login to do that